static/app/views/performance/browser/webVitals/pagePerformanceTable.tsx

```diff
@@ -82,7 +82,11 @@ export function PagePerformanceTable() {
     [projects, location.query.project]
   );
 
-  const sort = useWebVitalsSort();
+  let sort = useWebVitalsSort();
+  // Need to map fid back to inp for rendering
+  if (shouldReplaceFidWithInp && sort.field === 'p75(measurements.fid)') {
```

Comment:
This is a little messy right now. I plan to clean up a lot of conditionals in a separate pr. I think I'll find a more elegant way to fix this line in that follow up pr.